core/domain/user_domain.py

```diff
@@ -27,6 +29,10 @@
 (user_models,) = models.Registry.import_models([models.NAMES.user])
 
 
+SubjectIdUserIdPair = (
+    collections.namedtuple('SubjectIdUserIdPair', ['subject_id', 'user_id']))
```

Comment:
Can you call this firebase_subject_id or auth_subject_id? We are planning to use "subject" for things like Math/CS/etc.

Ditto elsewhere.

Reply:
Done, I just chose auth_id.